views/login-confirmation.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="{{ localeInfo.lang }}" dir="{{ localeInfo.direction }}">
+  <head>
+
+    <meta charset="utf-8">
+    {{ newrelic.getBrowserTimingHeader() | safe }}
+    <title>{{ gettext("Publish Your Remix") }}</title>
+
+    <link rel="stylesheet" href="/bower/makerstrap/dist/makerstrap.min.css" />
+    <link rel="stylesheet" href="/bower/webmaker-login-ux/dist/css/webmakerLogin.css">
+    <link rel="stylesheet" href="../stylesheets/style.css">
+    <link rel="stylesheet" href="../stylesheets/dialog.css">
+
+    <style>
+
+        h1 {
+          margin: 0.5em 0;
+        }
+
+        .dialog {
+          position: relative;
+        }
+
+        .dialog .body {
+          padding: 10%;
+          background-color: #fff;
+          text-align: center;
+        }
+
+        .dialog .body h1 {
+          margin-bottom: 3em;
+        }
+
+        .include-frame {
+          border: none;
+        }
+
+        .publish-enable-overlay {
+          display: none;
+        }
+
+        .webmaker-nav-container {
+          width: auto;
+          background-color: transparent;
+          box-shadow: none;
+          border: none;
+        }
+
+        .user-nav-container {
+          padding-top: 2rem;
+        }
+
+        #logo {
+          display: none;
+        }
+
+        .published-url {
+          word-break: break-all;
+          font-size: 200%;
+        }
+
+        #close {
+          position: absolute;
+          top: 1em;
+          right: 1em;
+          z-index: 10;
+        }
+
+    </style>
+  </head>
+  <body>
+
+    <div class="dialog">
+      <span class="button close-button fr" id="close">{{ gettext("Close") }}</span>
+
+      <link rel="stylesheet" href="stylesheets/userbar-overrides.css">
+
+      <header id="webmaker-nav">
+        <nav class="webmaker-nav-container">
+        </nav>
+      </header>
+
+      <div class="subheader">
+        <div class="pane">
+          <h2 class="subtitle">{{ gettext("Publish Your Remix") }}</h2>
+        </div>
+      </div>
+
+      <div class="body">
+        <p>You are now logged in, and can safely close this window</p>
+      </div>
+    </div>
+
+    <script id="require-js" data-csrf="{{ csrf }}"></script>
+
+    <script>
+      var tks = location.hash;
```

Comment:
see https://github.com/mozilla/id.webmaker.org/blob/develop/docs/oauth.md#2-webmaker-redirects-back-to-your-application-1